fix CV CurrentItem article (#1200)
telerik/maui-docs: controls/collectionview/current-item.md

The CollectionView allows you to use the `CurrentItem` property of type `object` to programmatically modify the current item during keyboard navigation, when using the mouse, and so on.

> The `RadCollectionViewItemView` exposes a read-only property `IsCurrent` (`bool`) which gets the value that indicates whether the view is visualized as current when navigating with the keyboard. The property returns `true` when the border to the current item gets visualized.

Reviewer: ```suggestion
The `RadCollectionViewItemView` exposes a read-only property `IsCurrent` (`bool`) which gets the value indicating whether the view is visualized as current during keyboard navigation. The property returns `true` when the border of the current item is visualized.
```
